src/OrchardCore.Modules/OrchardCore.Alias/Settings/AliasPartSettings.cs

```diff
@@ -4,7 +4,7 @@ namespace OrchardCore.Alias.Settings
 {
     public class AliasPartSettings
     {
-        [DefaultValue("{{ ContentItem.DisplayText | slugify }}")]
-        public string Pattern { get; set; } = "{{ ContentItem.DisplayText | slugify }}";
+        [DefaultValue("{{ Model.ContentItem.DisplayText | slugify }}")]
```

Comment:
Keep the defaults without Model

Reply:
As you want but i thought that the new rule, to be consistent, was to use the Model everywhere.

**And for compatibility i made it working in both cases, even if the actual settings doesn't use the Model**. Just re-tried it now works in both cases.

Let me know, otherwise i would have to also update what i did for autoroute, recipes ...